[WIP] Setting FileFieldGroup

New0 opened the pull request:
WIP: requires correct parameters and testing #9

New0 committed:
Setting FileFieldGroup

WIP: requires correct parameters and testing

New0 committed:
Starting tests for FileFieldGroup component

New0 committed:
WIP: Removed failing test, 

Writing all tests

New0 committed:
File field update value test

Shelob9 commented on src/components/fields/file-field/FileFieldGroup.js:
```diff
@@ -0,0 +1,104 @@
+					message={this.props.message}
+				/>
+			}
+				<label
```
Please use the FieldGroup.Label component

https://github.com/CalderaWP/caldera-components/blob/3684e109b6127a6d561aff9baddf3107f0f3a03c/src/components/fields/FieldGroup.js#L126

https://github.com/CalderaWP/caldera-components/blob/3684e109b6127a6d561aff9baddf3107f0f3a03c/src/components/fields/magic-select/MagicFieldGroup.js#L164-L167

New0 committed:
File field using FieldGroup type={'file'}

Display a file field using FieldGroup tag

New0 committed:
Correct FieldGroup text

New0 committed:
Pass FileFieldGroup functions to console log

New0 committed:
Make FileSelect a Dumb component

New0 committed:
Remove FileSelect tests

New0 committed:
FileFieldGroup props, label, propTypes

- Use FieldGroup.Label
- pass complete FileSelect props
- set propTypes in propTypes.js

New0 committed:
FileFieldGroup propTypes

New0 committed:
Further FileFieldGroup tests

New0 committed:
Change React file component to https://react-dropzone.js.org/

New0 committed:
Update props

New0 committed:
Tests snaps

New0 committed:
Frontend field test

New0 committed:
Regular input type="file" example

the example in app.js shows one way to use the regular input button for file upload

New0 commented:
https://github.com/CalderaWP/Caldera-Forms/pull/2775
@Shelob9 last commit shows a regular input type="file" field in the app.js examples
